### PR TITLE
Updated Blink Shell to list of iOS clients

### PR DESCRIPTION
**Pull request by carloscabanero**
Please see attached the changes to reference the iOS version on both the AppStore and GitHub.
Thanks!

### CODE REVIEW

**Comment by keithw** [comment]
Looks good! Can you please resubmit with passing tests? (The HTML5 validator is failing.)

File: index.html [hunk 1]
```diff
@@ -160,6 +160,7 @@ <h2 class="callout">Control-C works great.</h2>
   </div>
 
   <ul id="news-list">
+    <li><b>September 20, 2016</b>: <a href="http://blink.sh">Blink Shell: Mosh & SSH Terminal for iOS</a> has its first gold release on the <a href="http://itunes.apple.com/app/id1156707581">AppStore</a>. Free version is available on <a href="https://github.com/blinksh/blink">GitHub</a>.
```
Comment (andersk) [inline index.html]: `&` must be escaped as `&amp;` in HTML. Also, App Store is two words.

File: index.html [hunk 2]
```diff
@@ -287,6 +288,17 @@ <h3 class="callout"><a href="https://www.google.com/chrome"><img class="logo" sr
         Install <a href="https://chrome.google.com/webstore/detail/mosh/ooiklbnjmhbcgemelgfhaeaocllobloj">Mosh for Chrome.</a>
       </div>
     </div>
+
+    <div class="span4" style="vertical-align: top;">
+      <h3 class="callout"><img class="logo" src="App-store-icon.png" alt="">iOS</h3>
+      <div class="prelike">
+        <div style="float:left"><a href="http://blink.sh"><img src="blink_shell.png" alt="" style="height: 50px; width: 50px; display:inline-block; vertical-align:middle">
+        &nbsp;</a></div>
+	<a href="http://itunes.apple.com/app/id1156707581">Get Blink Shell from the AppStore</a><br/>
```
Comment (andersk) [inline index.html]: Should be HTTPS.

File: index.html [hunk 3]
```diff
@@ -287,6 +288,17 @@ <h3 class="callout"><a href="https://www.google.com/chrome"><img class="logo" sr
...
+	or...<br/> <a href="https://github.com/blinksh/blink">download from GitHub.</a>
+	<div style="clear:both"/>
```
Comment (andersk) [inline index.html]: `<div>` is not a void tag, so the self-closing `<div />` abbreviation cannot be used in HTML.

**Comment by carloscabanero** [comment]
Sorry about that, validated with html5 myself now too. Seems like all checks passed here too. Thanks @andersk for the corrections! :)

**Comment by cgull** [comment]
Pulled, thanks!  I think @keithw has to deploy this to the website manually.

**Comment by keithw** [comment]
Deployed to [mosh.org](https://mosh.org)!

**Pull request closed without merging**